docs/getting-started.rst

```diff
@@ -0,0 +1,427 @@
+Getting Started with OpenTelemetry Python
+=========================================
+
+This guide will walk you through instrumenting a python application with ``opentelemetry-python``.
+
+For more elaborate examples, see `examples`.
+
+Hello world: emitting a trace to your console
+---------------------------------------------
+
+To get started, install both the opentelemetry API and SDK:
+
+.. code-block:: sh
+
+    pip install opentelemetry-api
+    pip install opentelemetry-sdk
+
+The API package provides the interfaces required by the application owner, as well
+as some helper logic to load implementations.
+
+The SDK provides an implementation of those interfaces, designed to be generic and extensible enough
+that in many situations, the SDK will be sufficient.
+
+Once installed, we can now utilize the packages to emit spans from your application. A span
```

Comment:
Do we want to link to OT specs for people that reached this repo directly?

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/overview.md#span

Reply:
That sounds like a good idea to me, can you add the link as a suggestion in the PR (or push to this branch directly)?

Reply:
I'm not super familiar with ReStructuredText or how this will be rendered in readthedocs, so don't want to break this instead of improving it